tests/foreman/virtwho/test_ui_virtwho.py

```diff
@@ -134,3 +128,220 @@ def test_positive_deploy_configure_by_script(session, form_data):
         assert session.contenthost.search(guest_name)[0]['Subscription Status'] == 'green'
         session.virtwho_configure.delete(name)
         assert not session.virtwho_configure.search(name)
+
+
+@tier2
+def test_positive_debug_option(session, form_data):
+    """ Verify debug checkbox and the value changes of VIRTWHO_DEBUG
+
+    :id: d6f9940f-c003-4099-97bf-4d8820ab710e
+
+    :expectedresults:
+        1. if debug is checked, VIRTWHO_DEBUG=1 in /etc/sysconfig/virt-who
+        2. if debug is unchecked, VIRTWHO_DEBUG=0 in /etc/sysconfig/virt-who
+
+    :CaseLevel: Integration
+
+    :CaseImportance: Medium
+    """
+    name = gen_string('alpha')
+    form_data['name'] = name
+    with session:
+        session.virtwho_configure.create(form_data)
+        config_id = get_configure_id(name)
+        config_command = get_configure_command(config_id)
+        deploy_configure_by_command(config_command)
```

Comment:
Just a thought , can we add assertion for UI `Enable debugging output?` as well to make sure both(UI and file- /etc/sysconfig/virt-who ) end same reflect .

Reply:
It's not enough if we only validate the UI `debug` checked or uncheck, the significance of virtwhon configure is help to create the configure file in the host. After deploy the configure, it will change the /etc/sysconfig/virt-who, and create a configure file in /etc/virt-who.d/. 

So if without deployment and the validation , we are not sure the configure file can be working or not  which is created by the UI.

Reply:
Just clearing on my comment ...
Along with configuration validation(/etc/sysconfig/virt-who,) add validation at UI end as well ... so that we could see if UI and filesystem level has same thing or different ?

Reply:
@vijay8451  please review again with the new commit.